### PR TITLE
Fixes #4363 by reducing the default values for TASK_LOST handling.

### LINKED ISSUE #4363: Change defaults for TASK_LOST behavior
Default now: GC after 24h check every 1h initialDelay 5 min
Should be: GC after 75secs check every 30 secs initialDelay 5 min
Make a note in changelog about this breaking change

### PR DESCRIPTION
**Pull request by aquamatthias**

### CODE REVIEW

**Comment by aquamatthias** [comment]
Cherry pick in 1.3

File: changelog.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ Prior versions of Marathon have tried to authenticate whenever a principal has b
 Framework authentication is now explicit. There is a command line toggle option for authentication: `--mesos_authentication`.
 This toggle is disabled by default. You must now supply this flag to use framework authentication.
 
+#### Changed default values for TASK_LOST GC timeout
+If a task is declared lost in Mesos, Marathon waits for the task to come back for a certain amount of time.
```
Comment (meichstedt) [inline changelog.md]: Change to: `If a task is declared lost in Mesos, but the reason indicates it might come back, ...`

File: changelog.md [hunk 2]
```diff
@@ -22,6 +22,14 @@ Prior versions of Marathon have tried to authenticate whenever a principal has b
...
+To configure the behavior you can use `--task_lost_expunge_gc`, `--task_lost_expunge_initial_delay`, `--task_lost_expunge_interval`.
+Until Marathon 1.3 Marathon has handled TASK_LOST very conservatively: it waits for 24 hours for every task to come back.
```
Comment (meichstedt) [inline changelog.md]: `Until version 1.3, Marathon has handled ...`

**Comment by meichstedt** [comment]
Thanks @aquamatthias!

**Comment by meichstedt** [comment]
Cherry-picked into releases/1.3

**Pull request merged**